pkg/kubelet/status/state/state_checkpoint_test.go

```diff
@@ -0,0 +1,105 @@
+package state
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager"
+)
+
+func Test_stateCheckpoint_storeState(t *testing.T) {
+	cache := NewStateMemory()
+	checkpointManager, _ := checkpointmanager.NewCheckpointManager(".")
```

Comment:
I don't think source tree is a good place to keep temporary test artifacts. Please, consider using temporary directory, e.g. [as it's implemented for DRA checkpoint tests](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/dra/state/checkpointer_test.go#L153-L166)

Reply:
updated